libs/src/services/web3Manager/index.js

```diff
@@ -127,6 +127,19 @@ class Web3Manager {
     if (this.useExternalWeb3) {
       return ethSignTypedData(this.getWeb3(), this.getWalletAddress(), signatureData)
```

Comment:
Do we care about the external case?

Reply:
afaik nope! totally different call path. will test metamask signup as well though